bin/build-pouchdb.js

```diff
@@ -253,7 +255,7 @@ var doAll = argsarray(function (args) {
 });
 
 function doBuildNode() {
-  return mkdirp('lib/plugins')
+  return mkdirp(addPath('lib/plugins'))
```

Comment:
This was a mistake unrelated to this PR. We never noticed the mistake because you usually run `npm run build-node` after `npm run build` (or `install`, which runs `build`), in which case `packages/node_modules/pouchdb/lib/plugins` has already been `mkdirp`ed.